src/hooks/core/chat/useMoveChatTypeHook.ts

```diff
@@ -17,6 +17,7 @@ type ChannelCategory = 'SIGNED' | 'ANONYMOUS';
 interface MoveToChatChannelParams {
   targetUserId: string;
   oldChannelId: string;
+  source: 'userId';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `source` property in the `MoveToChatChannelParams` interface is currently hardcoded as `'userId'`. This reduces flexibility and could lead to incorrect usage. It would be better to define it as a string type, allowing different sources to be specified when needed.
```diff
-  source: 'userId';
+  source: string;
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->